lib/coverband/reporters/base.rb

```diff
@@ -30,19 +31,26 @@ def self.current_root
         File.expand_path(Coverband.configuration.root)
       end
 
-      protected
-
       def self.fix_file_names(report_hash, roots)
-        fixed_report = {} # normalize names across servers
-        report_hash.each_pair do |key, values|
+        if Coverband.configuration.verbose
+          Coverband.configuration.logger.info "fixing root: #{roots.join(', ')}"
+        end
+
+        fixed_report = {}
+        # normalize names across servers
+        report_hash.each_pair do |key, vals|
```

Comment:
Minor.. could get rid of the fixed_report local variable and just use each with object here?

```ruby
report_hash.each_with_object({}) do |(key, vals), fixed_report|
```

Reply:
done